src/containers/Manifest.js

```diff
@@ -36,6 +37,9 @@ const lifecycleMethods = {
     if (this.props.inMemoryData) {
       this.props.setInMemoryData(this.props.name, this.props.inMemoryData)
     } else {
+      if (this.props.filter && !this.props.filter.page) {
```

Comment:
why is there an if condition for count but not data?

Reply:
needs to be there